src/hooks/useStakingNFT.tsx

```diff
@@ -61,23 +61,12 @@ export const useStakingNFT = () => {
 			return;
 
 		const load = async () => {
-			try {
```

Comment:
Let's keep it as it was and calculate all rewards here. For the info for harvest modal on exit, we can make another call since it happens rarely.